bedrock/firefox/templates/firefox/pocket.html

```diff
@@ -0,0 +1,116 @@
+{# This Source Code Form is subject to the terms of the Mozilla Public
+ # License, v. 2.0. If a copy of the MPL was not distributed with this
+ # file, You can obtain one at http://mozilla.org/MPL/2.0/. -#}
+
+{% extends "firefox/base-resp.html" %}
+
+{% block page_title %}{{ _('Pocket') }}{% endblock %}
+{% block body_id %}firefox-pocket{% endblock %}
+{% block body_class %}
+  {{ super() }}
+  mozid
+{% endblock %}
+
+{% block page_css %}
+  {% stylesheet 'firefox_pocket' %}
+{% endblock %}
+
+{% block tabzilla_css %}{% endblock %}
+{% block tabzilla_js %}{% endblock %}
+
+{% block site_header %}
+    {% block tabzilla_tab %}
+    <div class="content">
+      <div id="tabzilla">
+        <a href="{{ url('mozorg.home') }}">Mozilla</a>
+      </div>
+    </div>
+    {% endblock %}
+    {% block site_header_nav %}{% endblock %}
+    {% block site_header_logo %}{% endblock %}
+    {% block breadcrumbs %}{% endblock %}
+{% endblock %}
+
+{% block content %}
+<main role="main">
+  <section class="content-main">
+    <div class="content">
+      <h1>{{ _('Pocket for Firefox') }}</h1>
+
+      <h2>
+        {{ _('Save it now. Read or watch it anytime, anywhere.') }}
+        <br>
+
+        {# Current Firefox #}
+        <span class="has-firefox">{{ _('All you need is your Firefox Account.') }}</span>
+
+        {# Old Firefox or Not Firefox #}
+        <span class="get-firefox">{{ _('Get Firefox to start using Pocket right in your browser.') }}</span>
+      </h2>
+
+      {# Has toolbar button #}
+      <p class="main-cta" id="cta-try"><button type="button" class="button-flat try-pocket">{{ _('Try it now') }}</button></p>
+
+      {# No toolbar button #}
+      <p class="main-cta" id="cta-learn"><a href="https://support.mozilla.org/kb/where-pocket-button-firefox" class="button-flat">{{ _('Learn more') }}</a></p>
+
+      {# Old Firefox or Not Firefox #}
+      <p class="main-cta" id="cta-download"><a href="{{ url('firefox.new') }}?scene=2#download-fx" class="button-flat-green">{{ _('Download Firefox') }}</a></p>
+
+    </div>
+  </section>
+
+  <section class="content-secondary">
+    <div class="content">
+      {# L10n: Pocket features below are emphasized with <strong></strong> #}
+      <ul>
+        <li>
+          <img src="{{ static('img/firefox/pocket/pocket-logo.svg') }}" width="122" height="122" alt="">
```

Comment:
It's really just IE8, and I was thinking it could live without these images but it shows the broken image placeholder, which does look pretty bad... I could fix it with CSS but then I need to do a separate style sheet just for IE8 and that's more overhead than I think it warrants. So I'll use your JavaScript trick to include PNG fallbacks.

Then again, I could just use PNG for all browsers with two resolutions, and that could be a CSS sprite and cut three requests down to one... but then it's not SVG and SVG is cool. They're presentational images, really, so maybe they belong in CSS anyway...

OK, I've talked myself into it. Switching CSS pseudo-elements and a PNG sprite.